registry-library/library/library.go

```diff
@@ -129,6 +145,16 @@ func GetRegistryIndex(registryURL string, options RegistryOptions, devfileTypes
 		endpoint = strings.TrimSuffix(endpoint, "&")
 	}
 
+	if options.NewIndexSchema && (options.Filter.MaxSchemaVersion != "" || options.Filter.MinSchemaVersion != "") {
+		if options.Filter.MinSchemaVersion != "" {
```

Comment:
I wonder if it would be better to add the query parameters like so:
```
req, err := http.NewRequest("GET", "<registry-url>", nil)
if err != nil {
    ...
}

q := req.URL.Query()
if options.Filter.MinSchemaVersion != "" {
    q.Add("minSchemaVersion", "options.Filter.MinSchemaVersion")
}
if options.Filter.MaxSchemaVersion != "" {
    q.Add("maxSchemaVersion", "options.Filter.MaxSchemaVersion")
}

req.URL.RawQuery = q.Encode()
...
```

Reply:
updated